api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-single/solutions-output/solutions-output.service.ts

```diff
@@ -21,6 +23,16 @@ export class SolutionsOutputService
     //
   }
 
+  interruptIfKilled = () => {
+    if (this.#cancelled) {
+      // TODO: do we need to cleanup anything?
+      throw {
+        stdError: [],
+        signal: 'SIGTERM',
+      };
+    }
+  };
+
   /**
    * @Alicia notes:
    * for the output files we will want to first run a small script in python that will do some calcs we need to do for the 5 most different solutions; and later we will need to save in the db prior a transformation the next files: output_sum.csv, outputmv_xxxx.csv and outputr_xxxx.csv
```

Comment:
unrelated to the topic of this PR, but this whole comment can be deleted now as the workflow we ended up implementing is different and this comment is not out of date and not relevant anymore.